.github/ISSUE_TEMPLATE/update_entry.md

```diff
@@ -1,10 +1,9 @@
 ---
-name: Update Entry
-about: Tell us if you found out that the deletion process changed on a given website
-title: ''
+name: Update Entry.
```

Comment:
```suggestion
name: Update Entry
```
No need for that in a title

Reply:
Beg to differ on that, but i can live with it.